torch/distributed/tensor/experimental/_attention.py

```diff
@@ -66,6 +71,29 @@ class _ContextParallelOptions:
 _cp_options = _ContextParallelOptions()
 
 
+@dataclass
+class _ContextParallelGlobalVars:
```

Comment:
Since we have this dataclass, should we move `_dispatch_mode` and `_cp_options` to this dataclass?

Reply:
Yes we can merge them. I'll do that in a follow-up PR (this requires torchtitan change as well).